addons/event_sale/models/event_registration.py

```diff
@@ -60,20 +60,17 @@ def _synchronize_so_line_values(self, so_line):
             }
         return {}
 
-    def summary(self):
-        res = super(EventRegistration, self).summary()
-        if self.event_ticket_id.product_id.image_128:
-            res['image'] = '/web/image/product.product/%s/image_128' % self.event_ticket_id.product_id.id
-        information = res.setdefault('information', {})
-        information.append((_('Name'), self.name))
+    def registration_summary(self):
+        res = super(EventRegistration, self).registration_summary()
+        information = res.get('registration').setdefault('information', {})
         information.append((_('Ticket'), self.event_ticket_id.name or _('None')))
```

Comment:
Same here.
Could append a "ticket_name" key and handle the rest at display level.

Backend should mostly be "display agnostic" and return base information.
The frontend takes care of displaying these information in an appropriate manner.